openedx/features/content_type_gating/templates/content_type_gating/access_denied_message.html

```diff
@@ -0,0 +1,19 @@
+<div class="content-paywall">
+    <div>
+        <h3>
+            <span class="fa fa-lock" aria-hidden="true"></span>
+            Verified Track Access
+        </h3>
+        <span style=" padding: 10px 0;">
+            Graded assessments are available to Verified Track learners.
+        </span>
+        {% if not mobile_app %}
+        <span class="certDIV_1" style="">
+            <a href="{{ecommerce_checkout_link}}" class="certA_2">
```

Comment:
Will this `ecommerce_checkout_link` always exist and be valid? Or should we add it to the `if` statement above?

Reply:
Fixed.